apps/sms/lib/bridge/plugins/contracts/service.js

```diff
@@ -0,0 +1,9 @@
+!((define)=>{define((require,exports,module)=>{
```

Comment:
I've manually removed this file from my PR, since we don't use it, but it's not a big deal anyway, just wanted to note :)